README.md

```diff
@@ -7,23 +7,29 @@ A browser extension (Chrome and Firefox) based on the https://github.com/wikiwho
 [![CircleCI](https://circleci.com/gh/wikimedia/WhoWroteThat.svg?style=svg)](https://circleci.com/gh/wikimedia/WhoWroteThat)
 
 ## Wikipedias
+
 The extension works only on Wikipedias supported by WhoColor API:
+
 * English Wikipedia: `*://en.wikipedia.org/*`
 * German Wikipedia: `*://de.wikipedia.org/*`
 * Basque Wikipedia: `*://eu.wikipedia.org/*`
 * Turkish Wikipedia: `*://tr.wikipedia.org/*`
 * Spanish Wikipedia: `*://es.wikipedia.org/*`
 
 ## Testing the Browser Extension
+
 There's a Grunt job to output the code into a working browser extension. To test it:
 
 1. Clone the repo
 2. Run `npm install`
-3. Run `./node_modules/.bin/grunt build`
+3. Run `npm build`
```

Comment:
Should be `npm run-script build` or `npm run build`. This may be dependent on the version of npm, not sure.

Reply:
Done.